src/System.Threading.Channels/src/System/Collections/Generic/Dequeue.cs

```diff
@@ -8,7 +8,7 @@ namespace System.Collections.Generic
 {
     /// <summary>Provides a double-ended queue data structure.</summary>
     /// <typeparam name="T">Type of the data stored in the dequeue.</typeparam>
-    [DebuggerDisplay("Count = {_size}")]
+    [DebuggerDisplay("Count = {_size}, Head={_head}, Tail={_tail}")]
```

Comment:
Are Head and Tail valuable here for someone hovering over the queue?  I wouldn't have guessed so, but maybe you have a scenario in mind?  I'd expect a more useful thing would be a debugger type proxy that showed the elements in an easily consumable manner, e.g. https://github.com/dotnet/corefx/blob/master/src/System.Collections/src/System/Collections/Generic/Queue.cs#L21

Reply:
I am seeing this would be nice to show when debugging a channel writer scenario when writing a new element and would like to see how the queue head/tail changes with the writing operation without going and browsing the queue details. 

I have to admit this is hypothetical scenario but I saw it helpful when looked at it under the debugger.

Reply:
But for the head and tail data to be useful, you'd also need to be looking at the array of data, right?  At which point you've already expanded the view and the tooltip is no longer useful?

Reply:
> But for the head and tail data to be useful, you'd also need to be looking at the array of data, right?

maybe not as I can see the capacity of the channel. the corner case scenario that I would be interested in is when the tail at the end of the buffer and I am trying to write extra element and want to see what will happen. in the local variable window, the channel object is showing the capacity and the queue will show the head and tail. for the tooltip, will need to hover over the channel to see the capacity. my point is, you'll not need to look at the expanded view. if you think this is an unlikely scenario, I am ok to remove it :-)

Reply:
I think this is unnecessary for anyone outside of someone implementing this code base.  Thanks.

Reply:
ok. I'll remove it then.